<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupDialog</class>
 <widget class="QDialog" name="GroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="sumWeightLabel">
     <property name="text">
      <string>Total weight</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sumWeightSpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="countLabel">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="countSpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="spreadLabel">
     <property name="text">
      <string>Spread</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spreadSpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="settingsLabel">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="keySpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="settingsSpinBox">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
